piet-cairo/Cargo.toml

```diff
@@ -10,12 +10,12 @@ keywords = ["graphics", "2d"]
 categories = ["rendering::graphics-api"]
 
 [dependencies]
-piet = { version = "0.2.0-pre5", path = "../piet" }
+piet = { version = "0.2.0-pre6", path = "../piet" }
```

Comment:
Did you mean "=0.2.0-pre6" here? Same is true for piet dep in the other back-ends.